MediaBrowser.Api/Playback/MediaInfoService.cs

```diff
@@ -86,6 +97,7 @@ public MediaInfoService(IMediaSourceManager mediaSourceManager, IDeviceManager d
             _userManager = userManager;
             _json = json;
             _authContext = authContext;
+            _logger = loggerFactory.CreateLogger("MediaInfoService");
```

Comment:
Again a nitpicking comment :hear_no_evil: `nameof(MediaInfoService)` instead? Would allow the logger to be renamed in case of a refactoring